sql-plugin/src/main/scala/com/nvidia/spark/rapids/SparkShims.scala

```diff
@@ -138,6 +139,8 @@ trait SparkShims {
     readFunction: (PartitionedFile) => Iterator[InternalRow],
     filePartitions: Seq[FilePartition]): RDD[InternalRow]
 
+  def getFileSourceMaxMetadataValueLength(sqlContext: SQLConf): Int
```

Comment:
I would rename sqlContext to be sqlConf here because you aren't passing the entire sqlContext in

Reply:
Done